### ISSUE TITLE
Implement blink character attribute

### ISSUE THREAD

**Issue by Tyriar** [xtermjs/xterm.js#944]
Being removed in https://github.com/sourcelair/xterm.js/pull/938. It would add additional complexity but maybe the cursor blink animation helper can be made generic and pulled into the `BaseRenderLayer` to simplify things.

**Comment by Tyriar** [comment]
Closing, no one cares about this and it's bad for a11y anyway.

**Comment by starball5** [comment]
Related on Stack Overflow: [Why don't ANSI escape codes for blinking text work in VS Code's integrated terminal?](https://stackoverflow.com/q/76535493/11107541)

**Comment by adontz** [comment]
> Closing, no one cares about this and it's bad for a11y anyway.

1. You cannot know all use cases of every software which uses xterm.js to make such statements. Honestly, it's kind of rude.
2. Specific use case is red blinking "PRODUCTION" in prompts in production environment. Since VS Code uses xterm.js, blinking does not work. VS Code is actually a very nice SSH client if one is focused on editing multiple configuration files.

**Comment by Tyriar** [comment]
@adontz few people care about this is more appropriate, sometimes my responses are short since I typically have 300-1000 issues on my plate at any given time.

Let's reopen as it would be good to implement it, we do parse and store the attribute already:

https://github.com/xtermjs/xterm.js/blob/d06d278f0ce69f6c7fdf4073e6b69d6973f96b47/typings/xterm.d.ts#L1740-L1741

It would likely be off by default with an option to enable though because of the a11y problems.

**Comment by quentinlamamy** [comment]
> > Closing, no one cares about this and it's bad for a11y anyway.
> 
> 1. You cannot know all use cases of every software which uses xterm.js to make such statements. Honestly, it's kind of rude.
> 2. Specific use case is red blinking "PRODUCTION" in prompts in production environment. Since VS Code uses xterm.js, blinking does not work. VS Code is actually a very nice SSH client if one is focused on editing multiple configuration files.

I agree with point 1. The logging and debugging system in my workspace uses blinking in specific cases. When you have multiple real-time terminals scrolling in front of you, blinking is an efficient way to alert the user when something critical happens.

Implement blink in xterm.js and i offer you coffee ☕️ :)

**Comment by Tyriar** [comment]
@quentinlamamy -> https://github.com/xtermjs/xterm.js/pull/5671, no coffee necessary 🙂

**Comment by Tyriar** [comment]
In VS Code this will be opt-in via `terminal.integrated.textBlinking` in the first 1.110 insiders coming soon: https://github.com/microsoft/vscode/pull/292685
